ConsumerApi/Controllers/VersionController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Backbone.BuildingBlocks.API.Mvc.ControllerAttributes;
+using Backbone.ConsumerApi.Mvc;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backbone.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+public class VersionController : ApiControllerBase
```

Comment:
You should't inherit from `ApiControllerBase`. It's not necessary in this case.

Reply:
Changes made.